tm12_amm:
  # 機器人採樣軌跡，笛卡爾座標 [x,y,z,rx,ry,rz]，旋轉角度單位為弧度
  sampleTrajectory:
    #- [0.214, -0.661, 0.500, -3.1416, 0.0000, 0.7854]#
    #- [0.308, -0.564, 0.500, -3.1416, 0.0000, 0.7854]
    #- [0.402, -0.466, 0.500, -3.1416, 0.0000, 0.7854]
    #- [0.496, -0.368, 0.500, -3.1416, 0.0000, 0.7854]
    #- [0.590, -0.270, 0.500, -3.1416, 0.0000, 0.7854]#
    #- [0.675, -0.355, 0.500, -3.1416, 0.0000, 0.7854]#
    #- [0.582, -0.453, 0.500, -3.1416, 0.0000, 0.7854]
    #- [0.488, -0.550, 0.500, -3.1416, 0.0000, 0.7854]
    #- [0.394, -0.647, 0.500, -3.1416, 0.0000, 0.7854]
    #- [0.300, -0.743, 0.500, -3.1416, 0.0000, 0.7854]#
    #- [0.385, -0.825, 0.500, -3.1416, 0.0000, 0.7854]#
    #- [0.479, -0.729, 0.500, -3.1416, 0.0000, 0.7854]
    #- [0.573, -0.633, 0.500, -3.1416, 0.0000, 0.7854]
    #- [0.667, -0.537, 0.500, -3.1416, 0.0000, 0.7854]
    #- [0.760, -0.440, 0.500, -3.1416, 0.0000, 0.7854]#
    #- [0.760, -0.440, 0.500, -3.1416, 0.0000, 0.7854]#
    - [0.655, -0.525, 0.400, -3.1416, 0.0000, 0.7854]
    - [0.561, -0.621, 0.400, -3.1416, 0.0000, 0.7854]
    - [0.467, -0.717, 0.400, -3.1416, 0.0000, 0.7854]
    #- [0.385, -0.825, 0.500, -3.1416, 0.0000, 0.7854]#
    #- [0.300, -0.743, 0.500, -3.1416, 0.0000, 0.7854]#
    - [0.394, -0.647, 0.400, -3.1416, 0.0000, 0.7854]
    - [0.488, -0.550, 0.400, -3.1416, 0.0000, 0.7854]
    - [0.582, -0.453, 0.400, -3.1416, 0.0000, 0.7854]
    #- [0.675, -0.355, 0.500, -3.1416, 0.0000, 0.7854]#
    #- [0.590, -0.270, 0.500, -3.1416, 0.0000, 0.7854]#
    - [0.508, -0.380, 0.400, -3.1416, 0.0000, 0.7854]
    - [0.414, -0.478, 0.400, -3.1416, 0.0000, 0.7854]
    - [0.321, -0.576, 0.400, -3.1416, 0.0000, 0.7854]
    #- [0.214, -0.661, 0.500, -3.1416, 0.0000, 0.7854]#
    #- [0.214, -0.661, 0.500, -3.1416, 0.0000, 0.7854]#
    - [0.348, -0.604, 0.300, -3.1416, 0.0000, 0.7854]
    - [0.442, -0.506, 0.300, -3.1416, 0.0000, 0.7854]
    - [0.536, -0.408, 0.300, -3.1416, 0.0000, 0.7854]
    #- [0.590, -0.270, 0.500, -3.1416, 0.0000, 0.7854]#
    #- [0.675, -0.355, 0.500, -3.1416, 0.0000, 0.7854]#
    - [0.582, -0.453, 0.300, -3.1416, 0.0000, 0.7854]
    - [0.488, -0.550, 0.300, -3.1416, 0.0000, 0.7854]
    - [0.394, -0.647, 0.300, -3.1416, 0.0000, 0.7854]
    #- [0.300, -0.743, 0.500, -3.1416, 0.0000, 0.7854]#
    #- [0.385, -0.825, 0.500, -3.1416, 0.0000, 0.7854]#
    - [0.439,-0.689, 0.300, -3.1416, 0.0000, 0.7854]
    - [0.533, -0.593, 0.300, -3.1416, 0.0000, 0.7854]
    - [0.627, -0.497, 0.300, -3.1416, 0.0000, 0.7854]
    #- [0.760, -0.440, 0.500, -3.1416, 0.0000, 0.7854]#
    - [0.615, -0.520, 0.145, -2.6180,  0.5236, 1.0472]
    - [0.535, -0.600, 0.120, -2.6180,  0.0000, 0.7854]
    - [0.450, -0.660, 0.110, -2.6180, -0.5236, 0.5236]
    - [0.415, -0.610, 0.150, -3.1416, -0.5236, 0.7854]
    - [0.360, -0.570, 0.165,  2.6180, -0.5236, 1.0472]
    - [0.415, -0.480, 0.190,  2.6180,  0.0000, 0.7854]
    - [0.505, -0.400, 0.205,  2.6180,  0.5236, 0.5236]
    - [0.555, -0.470, 0.195, -3.1416,  0.5236, 0.7854]
    - [0.580, -0.520, 0.225, -2.8798,  0.2618, 1.0472]
    - [0.505, -0.570, 0.140, -2.8798,  0.0000, 0.7854]
    - [0.445, -0.625, 0.185, -2.8798, -0.2618, 0.5236]
    - [0.450, -0.575, 0.150, -3.1416, -0.2618, 0.7854]
    - [0.420, -0.560, 0.210,  2.8798, -0.2618, 1.0472]
    - [0.450, -0.515, 0.190,  2.8798,  0.0000, 0.7854]
    - [0.485, -0.465, 0.250,  2.8798,  0.2618, 0.5236]
    - [0.515, -0.510, 0.195, -3.1416,  0.2618, 0.7854]
    - [0.385, -0.500, 0.220, -3.1416,  0.0000,-0.7854]
    - [0.480, -0.545, 0.150, -3.1416,  0.0000, 0.7854]
    - [0.530, -0.450, 0.220, -3.1416,  0.0000, 2.3562]
    - [0.480, -0.545, 0.200, -3.1416,  0.0000, 0.7854]
    - [0.480, -0.545, 0.250, -3.1416,  0.0000, 0.7854]
    - [0.480, -0.545, 0.300, -3.1416,  0.0000, 0.7854]

  # 棋盤格資訊
  chessboard_info:
    pattern_width: 11  # 棋盤格內角點數量
    pattern_height: 8
    square_size: 0.02     # 棋盤格方格大小（單位：m）